submissions/RemoveBG_Extension/action.js

```diff
@@ -0,0 +1,127 @@
+const apiKeys = ["rUQH1mhDaLKQe1ZpkKhFVjxL", "raSQ2Z22YVjDTkRAHBmQZErC"];
```

Comment:
i think you should find a way to hide this api keys to avoid abuse from someone else

Reply:
will i still need to hide if I don't care if its abused?

Reply:
I'll recommend you hide it. A simple solution would be to have an API that actually makes the API call and returns the image or something of the sort.